azure-pipelines.yml

```diff
@@ -29,13 +29,10 @@ stages:
         zivid_versions: ["${{ variables['newest_zivid_version'] }}"]
         ros_distros: ['ros:kinetic-ros-base-xenial']
         compilers: ['g++-7']
-#
-# Currently only newest_zivid_version is supported. Azure Pipelines requires at least one job under
-# jobs, so parameters.zivid_versions cannot be empty. Re-enable this when we support >0 old versions.
-#  - stage: Build_And_Test_Old_Zivid_Melodic_And_Kinetic
-#    jobs:
-#    - template: continuous-integration/azure-pipelines/templates/build_and_test.yml
-#      parameters:
-#        zivid_versions: []
-#        ros_distros: ['ros:melodic-ros-base-bionic', 'ros:kinetic-ros-base-xenial']
-#        compilers: ['g++-7']
+  - stage: Build_And_Test_Old_Zivid_Melodic_And_Kinetic
```

Comment:
This needs to remain commented out when we only support 1 SDK

Reply:
Fixed